### DISCUSSION TITLE
支持更多云盘：Onedrive，阿里云盘，123盘或者其他云盘。

### DISCUSSION THREAD

**Discussion by HumanMus1c** [rnchg/APT#4] [Ideas]
支持更多云盘：Onedrive，阿里云盘，123盘或者其他云盘。

**Comment by rnchg** [comment]
好的，回头再看看有没有好用的网盘，不过好像阿里云盘不能分享
________________________________
发件人: HumanMusic ***@***.***>
发送时间: 2024年7月8日 21:08
收件人: rnchg/Apt ***@***.***>
抄送: Subscribed ***@***.***>
主题: [rnchg/Apt] 支持更多云盘：Onedrive，阿里云盘，123盘或者其他云盘。 (Discussion #4)


支持更多云盘：Onedrive，阿里云盘，123盘或者其他云盘。

―
